lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -533,48 +530,55 @@ abstract class DefaultTextEditingStrategy implements TextEditingStrategy {
   @visibleForTesting
   bool isEnabled = false;
 
-  html.HtmlElement domElement;
-  InputConfiguration _inputConfiguration;
-  EditingState _lastEditingState;
+  html.HtmlElement get domElement => _domElement!;
+  html.HtmlElement? _domElement;
```

Comment:
Is this equivalent to:
```
late html.HtmlElement domElement;
```
?

Reply:
`domElement` is being reset to `null` in `disable()`, so can't use `late` here.